lib/operation.js

```diff
@@ -474,8 +474,10 @@ function negate (options) {
  * @param {Boolean} [normalise=true]
  * @returns {Sharp}
  */
-function normalise (normalise) {
+function normalise (normalise, lowerBin, upperBin) {
```

Comment:
How about having the API look a bit more like the following?
```js
.normalise({ lower: 1, upper: 99 })
```
(I'm happy to drop the `normalise` as-a-Boolean parameter as it's a little confusing.)

Reply:
[resolved](https://github.com/lovell/sharp/pull/3578/commits/8045250a2697591b40bef01db1c9c067587b1973)